Avoid automagic CXXFLAG detection courtesy of dev-libs/vc.

Gentoo-bug: 584118

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -617,10 +617,6 @@
         endif()
         endmacro()
 
-        if (NOT PACKAGERS_BUILD)
-        # Optimize the whole Calligra for current architecture
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${Vc_DEFINITIONS}")
-        endif ()
     endif()
     set(CMAKE_MODULE_PATH ${OLD_CMAKE_MODULE_PATH} )
 
